doc/modules/clustering.rst

```diff
@@ -1150,10 +1152,12 @@ following equation, from Vinh, Epps, and Bailey, (2009). In this equation,
 :math:`b_j = |V_j|` (the number of elements in :math:`V_j`).
 
 
-.. math:: E[\text{MI}(U,V)]=\sum_{i=1}^|U| \sum_{j=1}^|V| \sum_{n_{ij}=(a_i+b_j-N)^+
-   }^{\min(a_i, b_j)} \frac{n_{ij}}{N}\log \left( \frac{ N.n_{ij}}{a_i b_j}\right)
-   \frac{a_i!b_j!(N-a_i)!(N-b_j)!}{N!n_{ij}!(a_i-n_{ij})!(b_j-n_{ij})!
-   (N-a_i-b_j+n_{ij})!}
+.. math::
+  E[\text{MI}(U,V)]=\sum_{i=1}^{|U|} \sum_{j=1}^{|V|}
```

Comment:
No, that's not all I've done. For instance, the upper indices of the summations were absolute value bars in the original expression, which was obviously wrong.